docs/nosql/documentdb/one-to-many-relationship.md

```diff
@@ -0,0 +1,172 @@
+---
+id: one-to-many-relationship
+title: One to Many Relationship
+sidebar_label: One to Many Relationship
+---
+
+### One to many relationship (1:N)
+
+The 1:N relationship describes a relationship where one side can have more than one relationship 
+while the reverse relationship can only be single sided. 
+An example is a `Post` where a blog might have many `Comments` but a `Comment` is only related to a single `Post`.
+    
+
+
+### Embedded document pattern
+
+- Embedding connected data in a single document can reduce the number of read operations required to obtain data. 
+    In general, you should structure your schema so your application receives all of its required information in a single read operation.
+    If the number of embedded document(child documents) are limited to very less numbers then this can be better options.
+    For example, `User To Accounts`, `User To Addresses`, `Post To Comments`, can be modelled to embedded document pattern in one to many relationship.
+    
+    ```
+        {
+            "id": "1",
+            "name": "John Doe",
+            "accounts": [
+                {
+                    "bank": "123 fake bank",
+                    "accountNumber": "1231308239",
+                    "accountType": "saving",
+                    "address":"123 fake street",
+                    "createdDate":"2020-06-06",
+                    "valid" :true,
+                    "balance": "100.00",
+                },
+                {
+                    "bank": "abc fake bank",
+                    "accountNumber": "1231308239",
+                    "accountType": "saving",
+                    "address":"123 fake street",
+                    "createdDate":"2020-07-06",
+                    "valid" :true,
+                    "balance": "100.00",
+                }
+            ]
+        }
+
+    ```
+  
+ However, there are 3 potential problems associated with this approach that one should be aware off.
+ :::caution
+    - The embedded documents might grow larger and consequently growing main document size. NoSql DBs have limit on document size for example 16MB in case of Mongodb, 1 MB in case of google fire store.
+    - This could result in slow write performance as the embedded documents grows significantly larger in number.
+    - The third problem is exposed when one tries to perform pagination on the embedded documents 
+ :::
+  
+### Subset Pattern
```

Comment:
@RobusGauli  Yah, This is useful for properties that rarely change, that we can denormalize across multiple collections like name, dob.
And in the given example is properties like bank name or account number which won't change usually and can be duplicated across multiple collections to avoid joins